src/Module.zig

```diff
@@ -853,6 +856,14 @@ pub const File = struct {
         root: *Package.Module,
     };
 
+    pub fn modeFromPath(path: []const u8) Ast.Mode {
+        if (std.mem.endsWith(u8, path, ".zon")) {
+            return .zon;
+        } else if (std.mem.endsWith(u8, path, ".zig")) {
+            return .zig;
+        } else unreachable;
```

Comment:
I agree that this is unreachable, but it's probably worth mentioning why in a comment (`Module.importFile` will reject files without a `.zig`/`.zon` extension).